src/main/scala/io/citrine/lolo/bags/Bagger.scala

```diff
@@ -68,18 +73,22 @@ case class Bagger(
 
     /* Compute the number of instances of each training row in each training sample */
     val dist = new Poisson(1.0)(randBasis)
-    val Nib: Vector[Vector[Int]] = Iterator.continually{
-      // Generate Poisson distributed weights, filtering out any that don't have the minimum required number
-      // of non-zero training weights
-      Iterator.continually {
-        Vector.fill(trainingData.size)(dist.draw())
-      }.filter(_.count(_ > 0) >= Bagger.minimumNonzeroWeightSize).take(actualBags).toVector
-    }.filter{nMat =>
-      lazy val noAlwaysPresentTrainingData = nMat.transpose.forall{vec => vec.contains(0)}
-      // Make sure that at least one learner is missing each training point
-      // This prevents a divide-by-zero error in the jackknife-after-bootstrap calcualtion
-      !useJackknife || noAlwaysPresentTrainingData
-    }.next()
+    val Nib: Vector[Vector[Int]] = if (disableBootstrap) {
```

Comment:
I don't think this codepath is tested.  Its likely to cause some problems in the calculations below (e.g. `oobErrors` being empty) that can be handled.